coral-schema/src/main/java/com/linkedin/coral/schema/avro/ViewToAvroSchemaConverter.java

```diff
@@ -92,6 +92,57 @@ public Schema toAvroSchema(String dbName, String tableOrViewName) {
     return avroSchema;
   }
 
+  /**
+   * This is the main API to generate the avro schema String for a given Dali view
```

Comment:
This reads better: `An API to generate the Avro schema string for a view`.

Reply:
Could you change the other JavaDoc entries similarly?

Reply:
updated.